HIRS_ProvisionerTPM2/src/CommandTpm2.cpp

```diff
@@ -133,7 +134,8 @@ void CommandTpm2::setAuthData() {
     stringstream argsStream;
 
     switch (version) {
-        case Tpm2ToolsVersion::VERSION_1_1_0: {
+        case Tpm2ToolsVersion::VERSION_1_1_0:
+        case Tpm2ToolsVersion::VERSION_2_1_0: {
```

Comment:
Curly braces are not needed around the lines of a single case -- only to enclose the entire switch statement. Flow control is managed by jumping to the correct case and then leaving the switch block when hitting the `break` statement for the case.